sr_hand_health_report/src/sr_hand_health_report/motor_check.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+import rospy
+from sr_hand_health_report_check import SrHealthReportCheck
+from diagnostic_msgs.msg import DiagnosticArray
+
+
+class MotorCheck(SrHealthReportCheck):
+    def __init__(self, hand_side, fingers_to_test):
+        super().__init__(hand_side, fingers_to_test)
+        self._topic_name = '/diagnostics_agg'
+
+    def run_check(self):
+        result = {"motor_check": []}
+        try:
```

Comment:
This try seems a bit overkilled... especially for the exception message you want to send. Why not simply wrap the wait_for_message line only and catching the proper exception only?! https://docs.ros.org/en/diamondback/api/rospy/html/rospy.client-module.html#wait_for_message